content/integrations/awscloudtrail.md

```diff
@@ -3,6 +3,7 @@ title: Datadog-AWS CloudTrail Integration
 integration_title: AWS CloudTrail
 kind: integration
 newhlevel: true
+description: "Alert on suspicious AWS account activity."
```

Comment:
This doesn't seem like the right description.

Reply:
This is the desc on the corp website: 
![image](https://user-images.githubusercontent.com/3624790/31084478-cd2ce62c-a794-11e7-8734-c0f11c73fd4b.png)